examples/remoteBuild.md

```diff
@@ -59,20 +70,32 @@ test 3 == \
 
 The url should follow
 [hashicorp/go-getter URL format](https://github.com/hashicorp/go-getter#url-format).
-Here are some example urls pointing to Github repos following this convention.
 
-- a repo with a root level kustomization.yaml
+Note that S3 and GCS are NOT supported to avoid introducing massive dependency.
+
+Here are some example urls
+
+<!-- @createOverlay @testAgainstLatestRelease -->
+```
+DEMO_HOME=$(mktemp -d)
 
-  `github.com/Liujingfang1/mysql`
-- a repo with a root level kustomization.yaml on branch test
+cat <<EOF >$DEMO_HOME/kustomization.yaml
+resources:
 
-  `github.com/Liujingfang1/mysql?ref=test`
-- a subdirectory in a repo on version v1.0.6
+# a repo with a root level kustomization.yaml
+- github.com/Liujingfang1/mysql
 
-  `github.com/kubernetes-sigs/kustomize/examples/multibases?ref=v1.0.6`
-- a subdirectory in a repo on branch repoUrl2
+# a repo with a root level kustomization.yaml on branch test
```

Comment:
How about `github.com/kubernetes-sigs/kustomize/examples/multibases?ref=v1.0.6`?

Reply:
This is already covered by L91

```
# a subdirectory in a repo on branch repoUrl2
- github.com/Liujingfang1/kustomize/examples/helloWorld?ref=repoUrl2
```

Reply:
Already covered in L91 that followed. `kustomize` repository is too huge for testing. It takes a long time to clone when network is not so good.